transformer/request.py

```diff
@@ -66,6 +67,7 @@ def from_har_entry(cls, entry: dict) -> "Request":
             timestamp=pendulum.parse(entry["startedDateTime"]),
             method=HttpMethod[request["method"]],
             url=urlparse(request["url"]),
+            name=request["name"] if "name" in request else request["url"],
```

Comment:
Here, `request` is a [**HAR** _request_ object](http://www.softwareishard.com/blog/har-12-spec/#request), so it never contains a `name` field.
```suggestion
            name=None,
```